kin-sdk-tests/AccountInfoRetriverTest.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Threading.Tasks;
+using Kin.Sdk;
+using Kin.Base;
+
+namespace kin_sdk_tests
+{
+    [TestClass]
+    public class AccountInfoRetriverTest
+    {
+
+        static TestingKeystoreProvider keystoreProvider = new TestingKeystoreProvider();
```

Comment:
please use `TestInitialize` or `ClassInitialize` according to your needs
https://stackoverflow.com/a/1873572/3903847